src/main/java/com/softserve/controller/ArchiveController.java

```diff
@@ -47,11 +49,14 @@ public ResponseEntity archiveScheduleBySemester(@PathVariable("semesterId") Long
         Semester semester = semesterService.getById(semesterId);
         List<Schedule> schedules = scheduleService.getSchedulesBySemester(semesterId);
         if (schedules.isEmpty()) {
-            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new MessageDTO(String.format("Schedules with semesterId = %d not found.", semesterId)));
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new MessageDTO(String.format("Schedules with semesterId = %d not found.",
```

Comment:
it might be worth moving the body to a new line instead of semesterId only

Reply:
Okay